### PR TITLE
Introduce ReactiveAndroid

### PR DESCRIPTION
**Pull request by kittinunf**
### What's in this PR?

- [x] Introducing ReactiveAndroid (RA)

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -82,10 +83,16 @@ dependencies {
 
     compile "io.reactivex.rxjava2:rxjava:$rxJavaVersion"
     compile "io.reactivex.rxjava2:rxandroid:$rxAndroidVersion"
-    compile "com.jakewharton.rxbinding2:rxbinding:2.0.0"
```
Comment (worker8) [inline app/build.gradle]: ho ho.... ⚔️

File: app/build.gradle [hunk 2]
```diff
@@ -82,10 +83,16 @@ dependencies {
...
 
     compile "org.jetbrains.kotlin:kotlin-stdlib-jre7:$kotlinVersion"
 
+    compile "com.github.kittinunf.reactiveandroid:reactiveandroid-ui:$reactiveAndroidVersion"
+    compile "com.github.kittinunf.reactiveandroid:reactiveandroid-appcompat-v7:$reactiveAndroidVersion"
+
+    compile('com.crashlytics.sdk.android:crashlytics:2.7.1@aar') {
```
Comment (worker8) [inline app/build.gradle]: how about using `api` from gradle 3 so we don't need `transitive = true`

File: app/build.gradle [hunk 3]
```diff
@@ -11,9 +11,10 @@ ext {
 
     supportVersion = '26.1.0'
     constraintLayoutVersion = '1.0.2'
-    archComponentVersion = '1.0.0-rc1'
+    archComponentVersion = '1.0.0'
     rxJavaVersion = '2.1.6'
     rxAndroidVersion = '2.0.1'
+    reactiveAndroidVersion = '1.0.0-alpha3'
```
Comment (worker8) [inline app/build.gradle:17]: wonder why is it `alpha` still, what do you have in mind that is still lacking from this library❓

Comment (kittinunf) [reply app/build.gradle:17]: It is lacking tests! Also, some of the APIs are not fully migrated yet.

File: app/build.gradle (hunk 2 above)
Comment (kittinunf) [reply app/build.gradle]: 💡

File: app/build.gradle (hunk 1 above)
Comment (kittinunf) [reply app/build.gradle]: Who needs rxbinding ¯\_(ツ)_/¯ ?

**Comment by kittinunf** [comment]
マージ

**Pull request merged**